stock_putaway_product/wizard/stock_change_product_qty.py

```diff
@@ -13,7 +12,7 @@ class StockChangeProductQty(models.TransientModel):
     # right location to update stock.
     @api.model
     def default_get(self, fields):
-        res = super(StockChangeProductQty, self).default_get(fields)
+        res = super().default_get(fields)
```

Comment:
What means empty super?

Reply:
New in Python 3. It saves you to put class and instance